### DISCUSSION TITLE
Is it possible to specify multiple ssl listeners (on several interfaces)?

### DISCUSSION THREAD

**Discussion by nadav-bariks** [nanomq/nanomq#1342] [Q&A]
The documentation shows how to configure one listeners.ssl section with 1 bind 
which seems to support either one or all interfaces

is it possible to configure listeners on only 2 interfaces (if there are more than 2 on the computer)

thanks

**Comment by JaylinYu** [comment]
Not possible for now, but since you request, we will see how to plan this feature
